include stdint.h for int64 on musl (should be cstdint but gives an error related to C++11)

--- plugin-sysstat/lxqtsysstat.cpp	2015-02-08 11:31:27.000000000 -0200
+++ plugin-sysstat/lxqtsysstat.cpp	2015-07-28 23:51:15.864992133 -0300
@@ -37,6 +37,7 @@
 #include <QPainter>
 #include <QResizeEvent>
 #include <QVBoxLayout>
+#include <stdint.h>
 
 LxQtSysStat::LxQtSysStat(const ILxQtPanelPluginStartupInfo &startupInfo):
     QObject(),
